_docs/2-controlplane/1-controller/configuration-options.md

```diff
@@ -61,3 +62,52 @@ with a password, set the options above and this additional option as well:
 ```bash
 A8_DATABASE_PASSWORD=mypassword
 ```
+
+## Kubernetes Configuration
+
+Amalgam8 controller API serves as a frontend client to Kubernetes Third Party Resources (TPR), 
+ validating the rules and them storing.
+ Sample Kubernetes controller yaml file is found in `examples/k8s-controlplane.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: controller
+spec:
+  ports:
+  - port: 6080
+    targetPort: 8080
+    nodePort: 31200
+    protocol: TCP
+  selector:
+    name: controller
+  type: NodePort
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: controller
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: controller
+    spec:
+      containers:
+      - name: controller
+        image: amalgam8/a8-controller
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: A8_DATABASE_TYPE
+          value: kubernetes
```

Comment:
according etai, our naming convention has been `kubernetes` for similar environment variables